### PR TITLE
[PANW] Split field panw.panos.url_category_list in threat logs

### PR DESCRIPTION
**Pull request by kcreddy**
## What does this PR do?
This PR splits `panw.panos.url_category_list` field which contains comma-separated list of values.
Example:
`panw.panos.url_category_list: " ADFS Servers,computer-and-internet-info,low-risk" `

This PR adds a trim and a split processor on field `panw.panos.url_category_list`

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
```
$ elastic-package stack down && elastic-package build && elastic-package stack up --version=8.8.0 -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test pipeline --generate


2023/06/09 10:06:00  INFO New version is available - v0.81.0. Download from: https://github.com/elastic/elastic-package/releases/tag/v0.81.0
Detected shell: bash
2023/06/09 10:06:01  INFO New version is available - v0.81.0. Download from: https://github.com/elastic/elastic-package/releases/tag/v0.81.0
Run pipeline tests for the package
--- Test results for package: panw - START ---
╭─────────┬─────────────┬───────────┬──────────────────────────────────────────────┬────────┬──────────────╮
│ PACKAGE │ DATA STREAM │ TEST TYPE │ TEST NAME                                    │ RESULT │ TIME ELAPSED │
├─────────┼─────────────┼───────────┼──────────────────────────────────────────────┼────────┼──────────────┤
│ panw    │ panos       │ pipeline  │ test-panw-panos-authentication-sample.log    │ PASS   │   3.269666ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-config-sample.log            │ PASS   │     2.8135ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-correlated-events-sample.log │ PASS   │   2.265875ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-decryption-sample.log        │ PASS   │   3.855666ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-globalprotect-sample.log     │ PASS   │   8.091167ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-gtp-sample.log               │ PASS   │   3.464583ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-hipmatch-sample.log          │ PASS   │   3.495167ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-inc-other-sample.log         │ PASS   │   8.501792ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-inc-threat-sample.log        │ PASS   │  49.745625ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-inc-traffic-sample.log       │ PASS   │    49.6245ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-inc-traffic.json             │ PASS   │  13.872375ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-ip-tag-sample.log            │ PASS   │  11.248417ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-sctp-sample.log              │ PASS   │  11.647333ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-system-sample.log            │ PASS   │   3.298459ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-threat-sample.log            │ PASS   │ 101.998667ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-traffic-sample.log           │ PASS   │ 103.195041ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-tunnel-inspection-sample.log │ PASS   │   8.027542ms │
│ panw    │ panos       │ pipeline  │ test-panw-panos-userid-sample.log            │ PASS   │  17.936708ms │
╰─────────┴─────────────┴───────────┴──────────────────────────────────────────────┴────────┴──────────────╯
--- Test results for package: panw - END   ---
Done
```

## Related issues

- Closes https://github.com/elastic/integrations/issues/6519

### CODE REVIEW

File: packages/panw/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.11.0"
+  changes:
+    - description: Split panw.panos.url_category_list field in threat logs
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1111 # TODO
```
Comment (efd6) [inline packages/panw/changelog.yml:6]: DOTODO

**Review by efd6 — APPROVED** [review]
LGTM after changelog entry is up-to-date.

File: packages/panw/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.11.0"
+  changes:
+    - description: Split panw.panos.url_category_list field in threat logs
+      type: enhancement
```
Comment (bhapas) [inline packages/panw/changelog.yml:5]: bugfix ?

Comment (kcreddy) [reply packages/panw/changelog.yml:5]: In a way its also an `enhancement` as we are adding support to split the field 😄

Comment (bhapas) [reply packages/panw/changelog.yml:5]: Ok. Is this a new feature in threat logs by Palo Alto? Or did it exist before and we missed to implement. I would look at it this way to see if it qualifies an `enhancement` or `bugfix` and I think a patch step semantically informs the users that something is fixed which could be useful to upgrade to. Just my way of thinking though. WDYT?

Comment (kcreddy) [reply packages/panw/changelog.yml:5]: Does make sense. Although I am not sure if it existed before.

**Pull request merged**